css/main.css

```diff
@@ -343,6 +356,25 @@ input[type="number"]{
 	box-shadow: 0 4px 8px 0 rgba(0,0,0,0.1);
 }
 
+.warning::before{
+	content: '⚠️ ';
+	font-style: normal;
+	font-size: 1.2em;
+}
```

Comment:
this is in the css file twice?

Reply:
yeah idk what happened. I think maybe since I had to resolve conflicts I somehow kept both versions of that change or something. fixed it.